addons/website_sale/models/sale_order.py

```diff
@@ -235,6 +235,16 @@ def action_recovery_email_send(self):
             },
         }
 
+    @api.multi
+    def send_order_confirmation_mail(self, transaction=False):
```

Comment:
This code is very strange.  I don't understand why you provide two keys for website and not website sale teams with the same template by default. I think it should be simplified with one setting and one template. We don't care it comes from website or not; or it could be done in the template if people really want to separate things.